core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.ts

```diff
@@ -271,6 +272,37 @@ export class TutorCardComponent {
     return this.translateService.instant(milestoneMessageTranslationKey);
   }
 
+  shouldShowMilestoneProgressBar(): boolean {
```

Comment:
It seems like the method name does not completely capture the operations done. The current name seems to just suggest that some checks are performed and a boolean is returned, but looking at the implementation, it looks like we modify a component variable. A developer may use this method incorrectly (e.g. not knowing there is a side effect) in the future and can result in bugs. Can we rename this?

Reply:
Done